openprescribing/frontend/views/views.py

```diff
@@ -371,7 +371,11 @@ def analyse(request):
         # page load (see `alertForm` in `chart.js`)
         form = SearchBookmarkForm(
             initial={'email': getattr(request.user, 'email', '')})
-    return render(request, 'analyse.html', {'form': form})
+    context = {
+        'form': form,
+        'pcns_enabled': Practice.objects.filter(pcn_id__isnull=False).exists()
```

Comment:
Or `PCN.objects.active()`?

Reply:
That generates worse SQL and, given that this will be deleted anyway once we've imported PCN data, I'd rather leave it as is.

```sql
SELECT (1) AS "a" FROM "frontend_practice" WHERE "frontend_practice"."pcn_id" IS NOT NULL LIMIT 1
-- vs --
SELECT (1) AS "a" FROM "frontend_pcn" WHERE NOT ("frontend_pcn"."ons_code" IN (SELECT U0."ons_code" AS Col1 FROM "frontend_pcn" U0 LEFT OUTER JOIN "frontend_practice" U1 ON (U0."ons_code" = U1."pcn_id") WHERE U1."code" IS NULL)) LIMIT 1
```